packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -151,9 +151,9 @@ export interface DropTargetDelegate {
   getDropTargetFromPoint(x: number, y: number, isValidDropTarget: (target: DropTarget) => boolean): DropTarget | null
 }
 
-export interface DroppableCollectionProps {
+export interface UtilityOptions {
```

Comment:
separated the utility handlers of useDnDHooks so I could have them displayed separately in the docs

Reply:
This is in the shared types package, and this name is pretty general. `DroppableCollectionUtilityOptions`? Still not sure about the name "utility" but can't think of anything better.